arduino/smartcar.cpp

```diff
@@ -0,0 +1,184 @@
+#include <Smartcar.h>
```

Comment:
What is the difference between this and the smartcar.ino` file?

Reply:
It is a useless file, forgot to remove it before but it will get removed before the merge.  Accidentally added it here: https://github.com/DIT113-V22/group-02/commit/95abe68210d880c2a810a05f5186eedca7741dad#r72299274